crucible-llvm/src/Lang/Crucible/LLVM/MemModel/Generic.hs

```diff
@@ -1738,9 +1764,18 @@ ppMerge vpp c x y =
   where ppAllocList [] = (<+> text "<none>")
         ppAllocList xs = (<$$> indent 2 (vcat $ map vpp xs))
 
+ppAlignment :: Alignment -> Doc
```

Comment:
Nitpick: probably should special case for no particular alignment.  Seeing `1-byte-aligned` seems like it would b3e pretty confusing, as opposed to `no alignment` or just nothing at all.